content/en/getting-started/building-and-upgrading.md

```diff
@@ -27,12 +27,27 @@ As we saw in the previous section on how to configure Kairos, it is not possible
 Start by creating a file called `Dockerfile` with the following content
 
 ```
-FROM quay.io/kairos/ubuntu:24.04-standard-amd64-generic-v3.1.2-k3sv1.30.4-k3s1 
-
-RUN apt-get update && apt-get install -y ruby
+ARG BASE_IMAGE=ruby:3.4.4-bookworm # The ruby image is based on Debian bookworm, you can do the same with any of your images as long as they use a supported distro
```

Comment:
I would keep it simpler in here, especially it being the quick start. I would keep the from a published artifact and link to init below if they want a more hands on approach IMHO especially fro it to be a quick start, there is nothing faster than starting from an existing built artifact